src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsHelper.java

```diff
@@ -30,14 +33,24 @@ public static AmazonWebServicesCredentials getCredentials(@Nullable String crede
                 CredentialsMatchers.withId(credentialsId));
     }
 
+    private static boolean hasPermission(ItemGroup context) {
+        if (context instanceof Item) {
+            return ((Item) context).hasPermission(Item.CONFIGURE);
```

Comment:
```suggestion
            return ((Item) context).hasPermission(Item.EXTENDED_READ);
```

?

Reply:
Doesn't matter in practice since 2.223 or so.

Reply:
How so?

Reply:
The new read-only config form UI no longer renders actual drop-down fields where this would appear: https://github.com/jenkinsci/jenkins/pull/4479

(Unless Credentials Plugin is doing something weird in which case it should be adapted to support the new `readOnlyMode`.)

Reply:
@jglick do you agree moving this suggestion to another PR?

Reply:
This can be ignored.